backends/redis.go

```diff
@@ -31,79 +22,79 @@ type RedisConfig struct {
 	Hosts                 map[string]string
 	UseSSL                bool
 	SSLInsecureSkipVerify bool
+	Timeout               int
```

Comment:
See how it is implemented in the Gateway, Dash, or Pump, and do the same implementation. It has different arguments and also missing support for redis sentinel.